### ISSUE TITLE
Add read message for SQS Client

### ISSUE THREAD

**Issue by bararchy** [sdogruyol/aws#1]
It seems only `new` and `send_message` are available, is there a way to easily add `get` or `read` messages?

**Comment by marceloboeira** [comment]
Is the project still maintained @sdogruyol? I'm considering using crystal with a set of aws related apps and I'm happy to take over the project.

**Comment by sdogruyol** [comment]
@marceloboeira yes, it's still maintained. Any contribution is more than welcome :+1:

**Comment by leahpleurodon** [comment]
additionally, it would be nice if we could get a list of messages per queue :)

**Comment by tibbon** [comment]
This would be huge for me.
